controller/goal.controller.js

```diff
@@ -28,5 +29,54 @@ module.exports = {
             res.status(200).send(await ExperimentRepository.getVariantSuccessCount(id));
         else
             throw new PropertyNotFound("getVariantSuccessCountByExperimentID");
+    },
+
+    getAllGoals: async (req, res) => {
+        res.status(200).send(await GoalRepository.find());
+    },
```

Comment:
Do we need to get every goal in db? preferable to get goals by experiment id

Reply:
You are right, It's not a must and can be deleted on the approval of the PR.
Retrieving a single goal by Id is supported as well

Reply:
Great, so please remove this =]

Reply:
Removed